pennylane/plugins/default_qubit.py

```diff
@@ -121,23 +121,25 @@ def _apply_operation(self, operation):
         Args:
             operation (~.Operation): operation to apply on the device
         """
+        wires = operation.wires.tolist()  # TODO: translation to nonconsec wires indices
+
         if isinstance(operation, QubitStateVector):
-            self._apply_state_vector(operation.parameters[0], operation.wires)
```

Comment:
default.qubit was very recently refactored. hopefully we don't hit any merge conflicts

Reply:
So far all clean!